yb-voyager/cmd/export.go

```diff
@@ -408,12 +408,17 @@ func checkDependenciesForExport() (binaryCheckIssues []string, err error) {
 		missingTools = utils.CheckTools("strings")
 
 	case MYSQL:
-		// TODO: For mysql and oracle, we can probably remove the ora2pg check in case it is a live migration
-		// Issue Link: https://github.com/yugabyte/yb-voyager/issues/2102
-		missingTools = utils.CheckTools("ora2pg")
+		// In case if it is not a live migration, then we need to check for ora2pg
+		if !changeStreamingIsEnabled(exportType) && !useDebezium {
```

Comment:
nit: IMO, it's more readable to have `!(changeStreamingIsEnabled(exportType) || useDebezium)` . i.e. you can read it as if not (live migration OR offline with debezium)

Same for oracle case.

Reply:
Please test this manually. we don't have test-cases around this.